CardCarousel/CardCarousel.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react'
+import Arrow from './Arrow'
+
+export default class CardCarousel extends Component {
+  constructor(props) {
+    super(props)
+    this.state = ({
+      currentPracticeCardIndex: 0,
+      showAnswer: false,
+    })
+    this.hideAnswer = this.hideAnswer.bind(this);
+    this.nextImage = this.nextImage.bind(this);
+    this.previousImage = this.previousImage.bind(this);
+  }
+
+  previousImage() {
```

Comment:
Are we dealing with images here? Use the verbs and nouns from your "business domain". This is a flash card app. Scan your code for names that could be updated to better reflect that fact.